.eslintignore

```diff
@@ -0,0 +1 @@
+.eslintrc.js
```

Comment:
Can we make this not an extra file? :sweat_smile:

Reply:
Removed because eslintConfig is restored in `package.json`